src/createServer.js

```diff
@@ -1,3 +1,48 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase');
+const { handleErrorMessage } = require('./handleError');
+
+function createServer() {
+  const server = http.createServer((req, res) => {
+    res.setHeader('Content-Type', 'application/json');
```

Comment:
to make shorter your code you can create a separate function for this logic, because you send response in 2 places, but it's up to you